appgate/types.py

```diff
@@ -29,6 +29,8 @@ class OperatorArguments:
     timeout: str = attrib(default='30')
     cleanup: bool = attrib(default=True)
     target_tags: Optional[List[str]] = attrib(default=None)
+    builtin_tags: Optional[List[str]] = attrib(default=None)
```

Comment:
I would personally use factory=[] so you don't need the Optional type and the checks if None everywhere in the code.